frontend/src/components/TaskComponent.js

```diff
@@ -58,25 +53,29 @@ class Task extends Component {
             .catch(function (error) {
                 console.warn('Something went wrong', error)
```

Comment:
Use proper logging rather than logging to console.
[Logging Example in react](https://www.loggly.com/blog/best-practices-for-client-side-logging-and-error-handling-in-react/)